src/lib/ts-rlc-codegen/helpers/operationHelpers.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  MethodSignatureStructure,
+  OptionalKind,
+  ParameterDeclarationStructure
+} from "ts-morph";
+import { Methods, PathParameter } from "../interfaces";
+
+export function buildMethodDefinitions(
+  methods: Methods,
+  pathParams: PathParameter[] = []
+): OptionalKind<MethodSignatureStructure>[] {
+  const methodDefinitions: OptionalKind<MethodSignatureStructure>[] = [];
+  for (const key of Object.keys(methods)) {
+    const method = methods[key];
+    const description = methods[key][0].description;
+
+    let areAllOptional = methods[key][0].hasOptionalOptions;
+
+    methodDefinitions.push({
+      name: key,
+      ...(description && { docs: [{ description }] }),
```

Comment:
Is it guaranteed that those methods will always be documented? otherwise I am afraid undocumented ones will be flagged as undocumented by the API extractor tool making the review file less pleasant to read. Did you consider creating a default documentation for undocumented methods? maybe just the method name could work?